assignment/src/com/assignmentweek2/speedlimit/Honda.java

```diff
@@ -0,0 +1,32 @@
+package com.assignmentweek2.speedlimit;
+/*
+This is the Honda class
+ */
+public class Honda implements Bike{
+    private double speed;
```

Comment:
define a static final variable for speed. so that we can't modify its value after initializing once. Write a default method in interface to return speed.

Reply:
changed